tests/cases/fourslash/inlineVariableExportedVariable.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path="fourslash.ts" />
+
+////export const x/*a*/ = 1;
+////const y/*b*/ = 2;
+////export { y };

```

Comment:
Can you add a few other references to `x` and `y` that would be inlinable if not for the exports?

Reply:
Not sure I fully understand what you mean.

Reply:
Nvm I got it.